actions/hdl_example/sw/snap_example.c

```diff
@@ -624,7 +631,22 @@ int main(int argc, char *argv[])
 		case S121B_CARD:  VERBOSE1("S121B"); break;
 		case AD8K5_CARD:  VERBOSE1("AD8K5"); break;
 		case N250S_CARD:  VERBOSE1("N250S"); break;
-		case N250SP_CARD: VERBOSE1("N250SP"); break;
+		case N250SP_CARD:
+			/* N250SP can only handle 128 Byte Alignment and N*128 Bytes Size */
+			VERBOSE1("N250SP");
+			if (0 != (memcpy_align & 0x7F)) {
+				VERBOSE0("\nERROR: Can only handle 128 Byte alignment.\n");
+				VERBOSE0("       Adjust --align or -A to 128 or more\n");
```

Comment:
Is this hardcoded check going to be replaced by a different method, making use of the new bits in the SCaR?